crypto/evp/evp_enc.c

```diff
@@ -1051,6 +1051,48 @@ int EVP_CIPHER_CTX_ctrl(EVP_CIPHER_CTX *ctx, int type, int arg, void *ptr)
     return ret;
 }
 
+int EVP_CIPHER_get_params(EVP_CIPHER *cipher, OSSL_PARAM params[])
+{
+    if (cipher != NULL && cipher->get_params != NULL)
+        return cipher->get_params(params);
+    return 0;
+}
+
+int EVP_CIPHER_CTX_set_params(EVP_CIPHER_CTX *ctx, const OSSL_PARAM params[])
+{
+    if (ctx->cipher != NULL && ctx->cipher->ctx_set_params != NULL)
+        return ctx->cipher->ctx_set_params(ctx->provctx, params);
+    return 0;
+}
+
+int EVP_CIPHER_CTX_get_params(EVP_CIPHER_CTX *ctx, OSSL_PARAM params[])
+{
+    if (ctx->cipher != NULL && ctx->cipher->ctx_get_params != NULL)
+        return ctx->cipher->ctx_get_params(ctx->provctx, params);
+    return 0;
+}
+
+const OSSL_PARAM *EVP_CIPHER_gettable_types(const EVP_CIPHER *cipher)
+{
+    if (cipher != NULL && cipher->gettable_types != NULL)
```

Comment:
In my PR that overlaps this one - I have used the name that was used in the MAC's. We need to decide which naming convention you want and apply it to all of them..

Reply:
Er, the name change is entirely driven by what *others* want, FYI.